nautobot_ssot/integrations/infoblox/diffsync/models/nautobot.py

```diff
@@ -365,3 +375,19 @@ def delete(self):
         _vlan = OrmVlan.objects.get(id=self.pk)
         _vlan.delete()
         return super().delete()
+
+
+class NautobotNamespace(Namespace):
```

Comment:
Are we not expecting to do update or delete for this model? If so, we should have the methods and raise NotImplementedError.

Reply:
I'll add update. Not sure if it makes sense to implemente delete here.